src/tests/architect_tests/test_integration.py

```diff
@@ -367,22 +374,40 @@ def test_integration_simple():
         expected_matrix_multiplier=1,
     )
 
+    print(matrices_records)
+    assert len(matrices_records) == 4
+    assert matrices_records[0][0] == '8af84bde5a09e9adac63a384d924aab3' #The hash uuid
+    for n_examples in [row[1] for row in matrices_records]:
```

Comment:
This is another one that can be nuked from the individual tests and just go in `basic_integration_test`. It's true no matter what the test case.

Ideally, you shouldn't need to add any additional code to these test cases; all assertions should be in the common method or are too specific for the integration test.